### ISSUE TITLE
Automatic tmux start and automatic restore not working in OSX

### ISSUE THREAD

**Issue by chamoda** [tmux-plugins/tmux-continuum#19]
Automatic tmux start and automatic restore not working in OSX

Here's the .tmux.conf

```
unbind C-b
set-option -g prefix C-a
bind C-a send-prefix

set -g mouse-utf8 on
set -g mouse on

set-option -g default-command "reattach-to-user-namespace -l zsh"

set -g terminal-overrides 'xterm*:smcup@:rmcup@'

set -g default-command "reattach-to-user-namespace -l /bin/zsh"

# List of plugins
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-sensible'

# Other examples:
# set -g @plugin 'github_username/plugin_name'
# set -g @plugin 'git@github.com/user/plugin'
# set -g @plugin 'git@bitbucket.com/user/plugin'

# Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
run '~/.tmux/plugins/tpm/tpm'

set -g @plugin 'tmux-plugins/tmux-resurrect'
set -g @plugin 'tmux-plugins/tmux-continuum'

set -g @continuum-restore 'on'
set -g @continuum-boot 'on'
set -g @continuum-boot-options 'iterm'
```

tmux 2.1
iTerm2 2.9.20160313
OSX 10.11.4

Automatic saving is working fine.

**Comment by Leeiio** [comment]
Yes, i meet the same problem

**Comment by chamoda** [comment]
You can restore if you type `prefix + Ctrl-R` within 15 min of a tmux session start after restart. That's the workaround I'm using now

**Comment by Leeiio** [comment]
@chamoda Yes, prefix + ctrl-r can restore but automatic restore not working :(

**Comment by eduncan911** [comment]
It's always worked for me.  Here's my config on OSX:

**Comment by MareoRaft** [comment]
I have the same issue when I put the options BELOW the run-shell command.  I have the `lost server` issue if I put the options ABOVE the run-shell command.

**Comment by bruno-** [comment]
Sorry for the late reply everyone.

@chamoda , as for your `.tmux.conf` file in [this post](https://github.com/tmux-plugins/tmux-continuum/issues/19#issue-151722327), please ensure this line `run '~/.tmux/plugins/tpm/tpm'` is the last line in your `.tmux.conf`.

Everything else looks ok but we might have other bugs tho. My terminal of choice is `Terminal.app` and this one should work.

**Comment by bruno-** [comment]
@eduncan911 , I checked your `.tmux.conf` from [this post](https://github.com/tmux-plugins/tmux-continuum/issues/19#issuecomment-225644643) and it looks ok. Given that you're not using `tpm`, then yes, you have to run the plugins manually with `run-shell ...`

**Comment by bruno-** [comment]
I'm not sure what's causing the `lost server` error ..

**Comment by rofrol** [comment]
Maybe related https://github.com/tmux-plugins/tmux-continuum/issues/52#issuecomment-44507
